docs/source/_toctree.yml

```
@@ -13,6 +13,8 @@
     title: Choosing the right metric
   - local: creating_and_sharing
     title: Adding new evaluations
+  - local: metric_integrations
+    title: Using 🤗 Evaluate in other frameworks
```

```suggestion
    title: Using 🤗 Evaluate with other ML frameworks
```
not a strongly held suggestion here, take it or leave it